addons/mail/static/src/discuss/call/common/call_settings.xml

```diff
@@ -40,17 +40,19 @@
                     <div class="mb-3 d-flex align-items-center flex-wrap">
                         <label class="d-flex align-items-center flex-wrap mw-100 cursor-pointer" title="Delay after releasing push-to-talk" aria-label="Delay after releasing push-to-talk">
                             <span class="me-2 text-truncate">Delay after releasing push-to-talk</span>
-                            <div class="d-flex w-100">
-                            <input class="flex-grow-2 form-range" type="range" min="200" max="2000" step="1" t-att-value="userSettings.voiceActiveDuration" t-on-change="onChangeDelay"/>
+                            <div class="d-flex w-100 align-items-center">
+                                <input class="flex-grow-2 form-range" type="range" min="200" max="2000" step="1" t-att-value="userSettings.voiceActiveDuration" t-on-input="onChangeDelay"/>
+                                <span class="p-1"><t t-out="userSettings.voiceActiveDuration"/>ms</span>
                             </div>
                         </label>
                     </div>
                 </t>
                 <div t-else="" class="mb-3 d-flex align-items-center flex-wrap">
                     <label class="d-flex align-items-center flex-wrap mw-100 cursor-pointer" title="Voice detection threshold" aria-label="Voice detection threshold">
                         <span class="me-2 text-truncate">Voice detection threshold</span>
-                        <div class="d-flex w-100">
-                            <input class="flex-grow-2 form-range" type="range" min="0.001" max="1" step="0.001" t-att-value="userSettings.voiceActivationThreshold" t-on-change="onChangeThreshold"/>
+                        <div class="d-flex w-100 align-items-center">
+                            <input class="flex-grow-2 form-range" type="range" min="0.001" max="1" step="0.001" t-att-value="userSettings.voiceActivationThreshold" t-on-input="onChangeThreshold"/>
+                            <span class="p-1"><t t-out="Math.floor(userSettings.voiceActivationThreshold * 100)"/>%</span>
```

Comment:
Minor but:
The size of the number conflict with the size of the range slider.
Easy fix would be to make the span a fixed with?